### PR TITLE
Epmlstrtai 732 5 user update role test

### PR DESCRIPTION
**Pull request by Sadredneck**
Role change test created

### CODE REVIEW

File: jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/container/TestContext.java [hunk 1]
```diff
@@ -15,6 +15,10 @@
 
     private String token;
 
+    private String adminToken;
+
+    private int counter=0;
```
Comment (AleksandrKugushev) [inline jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/container/TestContext.java]: Could you fix spacing?

File: jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/container/TestContext.java [hunk 2]
```diff
@@ -49,4 +53,20 @@ public String getToken() {
     public void setToken(String token) {
         this.token = token;
     }
-}
+
+    public int getCounter() {
+        return counter++;
```
Comment (AleksandrKugushev) [inline jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/container/TestContext.java]: A getter shouldn't have side effects. I suggest changing name or behavior

File: jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/given/Authentication.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import com.epam.test_generator.api.steps.StepBackground;
+import com.epam.test_generator.controllers.user.request.LoginUserDTO;
+import cucumber.api.java.en.Given;
+
+public class Authentication extends StepBackground {
+
+    @Given("^I am Admin$")
+    public void logAdmin(){
+        LoginUserDTO loginUserDTO = new LoginUserDTO();
+        loginUserDTO.setEmail("admin@mail.com");
```
Comment (AleksandrKugushev) [inline jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/given/Authentication.java]: Let's stop using magic strings. I suggest creating a final field to store "admin@mail.com" and reuse it,

Comment (Sadredneck) [reply jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/given/Authentication.java]: Constands created

File: jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/container/TestContext.java (hunk 2 above)
Comment (Sadredneck) [reply jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/container/TestContext.java]: Added getCounterAndIncrease, could be removed in future

File: jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/container/TestContext.java (hunk 1 above)
Comment (Sadredneck) [reply jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/container/TestContext.java]: Fixed

**Pull request closed without merging**